htdocs/core/class/extrafields.class.php

```diff
@@ -2086,7 +2086,7 @@ public function setOptionalsFromPost($extralabels, &$object, $onlykey = '')
 						$value_key = '';
 					}
 				}
-				elseif (in_array($key_type, array('price', 'double')))
+				elseif (in_array($key_type, array('price', 'double', 'html')))
```

Comment:
You apply the method price2num on a html field ?

Reply:
Sorry, looked around the price2num